funfact/lang/_tsrex.py

```diff
@@ -415,7 +416,7 @@ def indices(spec):
         raise RuntimeError(f'Cannot create indices from {spec}.')
 
 
-def tensor(*spec, initializer=None, optimizable=None):
+def tensor(*spec, initializer=None, optimizable=None, prefer=NoCondition()):
```

Comment:
Similar here, `prefer=None` might be a more readable approach.

Reply:
Changed the default argument to `None`. Added
```   
 if optimizable and prefer is None:
        prefer = NoCondition()
```

as we need this for the `vmap` of optimizable leafs. The penalty should be a 1D array after vmap.